phpmyfaq/src/phpMyFAQ/Setup/Installer.php

```diff
@@ -324,6 +324,7 @@ class Installer extends Setup
         'search.enableRelevance' => 'false',
         'search.enableHighlighting' => 'true',
         'search.searchForSolutionId' => 'true',
+        'search.popularSearchTimeWindow' => '180',
```

Comment:
this new configuration needs an entry in the language files, at least for English and German